pages/linux/lrunzip.md

```diff
@@ -0,0 +1,24 @@
+# lrunzip
+
+> A large file decompression program.
+> See also `lrzip`, `lrztar`, `lrzuntar`.
+
+- Decompress a file:
+
+`lrunzip {{filename.lrz}}`
+
+- Decompress a file using more processor threads:
```

Comment:
Perhaps ....`using a specific number of processor threads` would be more accurate?

Reply:
That's all set!